jormungandr/src/blockchain/chain.rs

```diff
@@ -50,6 +50,7 @@ See Internal documentation for more details: doc/internal_design.md
 */
 
 use super::{branch::Branches, reference_cache::RefCache};
+use crate::blockchain::StorageError;
 use crate::{
     blockcfg::{
         Block, Block0Error, BlockDate, ChainLength, Epoch, EpochRewardsInfo, Header, HeaderHash,
```

Comment:
if you are to more the import, put it in the same _scope_ of all the `use crate::{*}` just below

Reply:
done